hack/release.go

```diff
@@ -107,9 +126,13 @@ func main() {
 	templateDir := flag.String("templateDir", ".", "Directory with templates to render")
 	outputDir := flag.String("outputDir", ".", "Destination directory to render templates to")
 	releaseDefaults := flag.String("releaseDefaults", "release.yaml", "Destination of the release defaults definition")
+	retrieveSha := flag.Bool("with-sha256", false, "retrieve SHA256 for container images")
```

Comment:
nit: retrieve SHA256 for container image references

Reply:
Done